{"name": "test_post_and_verify_message", "status": "failed", "statusDetails": {"message": "AssertionError: assert 404 == 200\n +  where 404 = <Response [404]>.status_code", "trace": "@pytest.mark.asyncio\n    async def test_post_and_verify_message():\n        session_id = str(uuid.uuid4())\n        headers = {\n            \"Cookie\": f\"session-id={session_id}\"\n        }\n        message_data = {\n            \"id\": \"unique-id\",\n            \"ts\": 0,\n            \"text\": \"Test message\"\n        }\n    \n        # Отправка POST-запроса\n        post_response = requests.post(BASE_URL, headers=headers, json=message_data)\n>       assert post_response.status_code == 200\nE       assert 404 == 200\nE        +  where 404 = <Response [404]>.status_code\n\ntests/backend/test_api.py:21: AssertionError"}, "start": 1734533072891, "stop": 1734533073141, "uuid": "8cc79e5d-abbf-460c-aabe-bb1a687665fa", "historyId": "dfdfece5aa80aac29b567406bace4c1f", "testCaseId": "dfdfece5aa80aac29b567406bace4c1f", "fullName": "tests.backend.test_api#test_post_and_verify_message", "labels": [{"name": "tag", "value": "asyncio"}, {"name": "parentSuite", "value": "tests.backend"}, {"name": "suite", "value": "test_api"}, {"name": "host", "value": "Mac"}, {"name": "thread", "value": "55493-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.backend.test_api"}]}